Framework/PythonInterface/test/python/plugins/algorithms/CalculateSampleTransmissionTest.py

```diff
@@ -13,16 +13,16 @@ def test_sample_transmission_calculation(self):
 
         # Using water sample
         formula = "H2-O"
-        density = 0.033424
+        density = 0.100272
```

Comment:
What's the reason for changing the value here? Should this test case not stay the same?